src/main/kotlin/org/jitsi/nlj/rtp/TransportCcEngine.kt

```diff
@@ -165,8 +164,8 @@ class TransportCcEngine(
                 } else {
                     "Latest seqNum was ${sentPacketDetails.lastSequence}, size is ${sentPacketDetails.size}."
                 } +
-                if (lastRtt != UNKNOWN) {
-                    " Latest RTT is ${lastRtt.formatMilli()} ms."
+                if (lastRtt != null) {
+                    " Latest RTT is ${lastRtt!!.formatMilli()} ms."
                 } else {
                     ""
                 })
```

Comment:
I think this could also be:
```
...
} +
lastRtt?.let {
  " Latest RTT is ${it.formatMilli()} ms."
} ?: run { 
  ""
}
```
which avoids the `!!`